packages/nimble-components/src/rich-text/models/configuration.ts

```diff
@@ -0,0 +1,25 @@
+import type { RichTextMention } from '../../rich-text-mention/base';
+import { MarkdownParserMentionConfiguration } from './markdown-parser-mention-configuration';
+
+/**
+ * Base class for Configuration, contains mention configuration required for RichTextMarkdownParser
+ */
+export class Configuration {
+    public parserMentionConfig: MarkdownParserMentionConfiguration[] = [];
+
+    protected isValid = true;
+
+    public constructor(mentionElements: RichTextMention[]) {
+        this.isValid = mentionElements.every(
+            mention => mention.mentionInternals.validConfiguration
```

Comment:
So many similar things called mention, lets be moree explicit

```suggestion
            mentionElement => mentionElement.mentionInternals.validConfiguration
```

Reply:
Updated